app/controllers/support_interface/validation_errors_controller.rb

```diff
@@ -1,33 +1,5 @@
 module SupportInterface
   class ValidationErrorsController < SupportInterfaceController
-    def index
-      @grouped_counts = ValidationError.group(:form_object).order('count_all DESC').count
-      @grouped_column_error_counts = ValidationError.list_of_distinct_errors_with_count
-    end
-
-    def search
-      @validation_errors = ValidationErrorSearch
-        .search(params)
-        .includes('user')
-        .order('created_at DESC')
-        .page(params[:page] || 1)
-    end
-
-    def summary
-      sort_param = params.permit(:sortby)[:sortby]
-
-      @validation_error_summary = ::ValidationErrorSummaryQuery.new(sort_param).call
-    end
-
-    class ValidationErrorSearch
```

Comment:
I moved this to the `ValidationError` class as felt suitable to live there as well as test